obi_one/scientific/simulation/stimulus.py

```diff
@@ -349,3 +356,85 @@ def _generate_config(self) -> dict:
 
 class SynchronousSingleSpikeStimulus(Stimulus):
     spike_probability: float | list[float]
+
+
+class SpikeStimulus(Stimulus):
+    _module: str = "synapse_replay"
+    _input_type: str = "spikes"
+    duration: float
+    gid_spike_map: dict = {}
+    spike_file: Path | None = None
```

Comment:
I think this variable could be made private, since it is only set and used internally. Also, in this way, it won't be part of the (de-)serialization. (Will require changing all occurrences!)
```suggestion
    _spike_file: Path | None = None
```